README.md

```diff
@@ -1,2 +1,242 @@
+    pr-numer: 109
+    location: "body"
+    title: "[Rr]elease [Nn]otes:"
+    skip-labels: "skip-release-notes,no-release-notes"
+    fails-on-error: "false"
+  ```
+
+## Running Static Code Analysis
+
+This project uses Pylint tool for static code analysis. Pylint analyses your code without actually running it. It checks for errors, enforces, coding standards, looks for code smells etc.
+
+Pylint displays a global evaluation score for the code, rated out of a maximum score of 10.0. We are aiming to keep our code quality high above the score 9.5.
+
+### Set Up Python Environment
+```shell
+python3 -m venv venv
+source venv/bin/activate
+pip install -r requirements.txt
+
+# run your commands
+
+deactivate
+```
+
+This command will also install a Pylint tool, since it is listed in the project requirements.
+
+### Run Pylint
+Run Pylint on all files that are currently tracked by Git in the project.
+```
+pylint $(git ls-files '*.py')
+```
+
+To run Pylint on a specific file, follow the pattern `pylint <path_to_file>/<name_of_file>.py`.
+
+Example:
+```
+pylint ./release_notes_presence_check/release_notes_presence_check_action.py
+``` 
+
+## Run Black Tool Locally
+This project uses the [Black](https://github.com/psf/black) tool for code formatting.
+Black aims for consistency, generality, readability and reducing git diffs.
+The coding style used can be viewed as a strict subset of PEP 8.
+
+The project root file `pyproject.toml` defines the Black tool configuration.
+In this project we are accepting the line length of 120 characters.
+
+Follow these steps to format your code with Black locally:
+
+### Set Up Python Environment
+From terminal in the root of the project, run the following command:
+
+```shell
+python3 -m venv venv
+source venv/bin/activate
+pip install -r requirements.txt
+```
+
+This command will also install a Black tool, since it is listed in the project requirements.
+
+### Run Black
+Run Black on all files that are currently tracked by Git in the project.
+```shell
+black $(git ls-files '*.py')
+```
+
+To run Black on a specific file, follow the pattern `black <path_to_file>/<name_of_file>.py`.
+
+Example:
+```shell
+black ./release_notes_presence_check/release_notes_presence_check_action.py
+``` 
+
+### Expected Output
+This is the console expected output example after running the tool:
+```
+All done! ✨ 🍰 ✨
+1 file reformatted.
+```
+
+## Running Unit Test
+
+Unit tests are written using pytest. To run the tests, use the following command:
+
+```
+pytest tests/
+```
+
+This will execute all tests located in the tests directory.
+
+## Code Coverage
+
+Code coverage is collected using pytest-cov coverage tool. To run the tests and collect coverage information, use the following command:
+
+```
+pytest --cov=. --cov-report=html tests/
+```
+
+This will execute all tests located in the tests directory and generate a code coverage report.
+
+See the coverage report on the path:
+
+```
+htmlcov/index.html
```

Comment:
There are many commands, that are missing `shell` after ```. I suggest to add the shell format for the highlighted blocks of code. In this particularly I would like to see as a user command `open htmlcov/index.html`, which will open the cov html page in my browser.

Reply:
Fixed in commit